zinc/src/test/scala/sbt/inc/LookupSpec.scala

```diff
@@ -93,17 +67,19 @@ class LookupSpec extends BaseIncCompilerSpec {
     lookup.lookupAnalysis("P1$P2") shouldEqual None
   }
 
-  it should "find correct analysis for class file" in mockedProjectContext { setup =>
-    import setup._
+  // it should "find correct analysis for class file" in mockedProjectContext { setup =>
+  //   import setup._
 
-    lookup.lookupAnalysis(project1.P1P2.classFile) shouldEqual project1.analysis
-    lookup.lookupAnalysis(project2.P1P2.classFile) shouldEqual project2.analysis
-    lookup.lookupAnalysis(project2.P2.classFile) shouldEqual project2.analysis
-    lookup.lookupAnalysis(project1.P1J2.classFile) shouldEqual project1.analysis
-    lookup.lookupAnalysis(project1.P1C2.classFile) shouldEqual project1.analysis
+  //   val lookup = new LookupImpl(config)
```

Comment:
I don't think 9dabad4 is a proper fix. PerClasspathEntryLookup is documented to work with cp entry but Lookup should work with classes. 

I think we should revert only lines 35-36 in LookupImpl.scala from my implementation.